### DISCUSSION TITLE
Additional Features

### DISCUSSION THREAD

**Discussion by camrun91** [ApollosProject/apollos-templates#377] [Q&A]
On the latest version, it looks like the home tab has been moved into the index.js in the tabs folder. How do I pass in the additional features that I was passing in using the additionalFeatures Prop?

**Comment by redreceipt** [comment]
Good question, copying @vinnyjth

**Comment by camrun91** [comment]
I will say I like how the change cleans up the API config.json file.

**Comment by vinnyjth** [comment]
I'll link you a code example! 

https://github.com/NewSpring/corinth/blob/master/src/theme/propOverrides.js#L89-L91

This is the best way to do it, since it will auto-magically work with all the feature feeds in your app, and will survive any apollos upgrades.

**Reply by camrun91** [reply]
When I do it this way I am getting an invalid hook call error. I am going to look into that to see what is causing it.

**Comment by camrun91** [comment]
@vinnyjth so doing it the way that you suggested throws an invalid hook error. However if I add `tabProps: {
    additionalFeatures: {
      LiveStreamListFeature: LiveStreamListFeatureConnected,
    },
  },`

Then it works as expected.  I assume the risk of doing it this way is that an update could change it?

**Reply by vinnyjth** [reply]
@camrun91 I think the risk is more that it will only work on the `TabFeatureFeeds` and not on other feature feeds. For your use case though, I think you're good! The hook error is weird though... any chance you have multiple versions of `@apollo/client` installed?

**Reply by camrun91** [reply]
I mean I guess there is always a chance. I dk how that would have happened though.
